src/main/kotlin/step2/StringCalculator.kt

```diff
@@ -0,0 +1,35 @@
+package step2
+
+private const val INPUT_NULL_ERROR_MESSAGE = "입력 값은 null일 수 없습니다."
+
+private const val INPUT_BLANK_ERROR_MESSAGE = "입력 값은 공백일 수 없습니다."
+
+private const val OPERATOR_ERROR_MESSAGE = "올바른 사칙연산 기호를 입력해주세요."
+
+private const val DELIMITER = " "
+
+class StringCalculator {
+
+    fun calculate(input: String?): Int {
+        val parseList = parse(input)
+        var result = parseList[0].toInt()
+        for (i in 1 until parseList.size step (2)) {
+            val num = parseList[i + 1].toInt()
+            val operator = Operator.from(parseList[i]) ?: throw IllegalArgumentException(OPERATOR_ERROR_MESSAGE)
+            result = operator.calculate(result, num)
+        }
+        return result
+    }
+
+    private fun parse(input: String?): List<String> {
+        if (input == null) {
+            throw IllegalArgumentException(INPUT_NULL_ERROR_MESSAGE)
+        }
+
+        if (input.isBlank()) {
+            throw IllegalArgumentException(INPUT_BLANK_ERROR_MESSAGE)
+        }
```

Comment:
아래와 같이 null 또는 Blank인 경우 true를 반환하는 메서드를 대신 활용해볼 수 있을 것 같아요. 😃
```suggestion
        if (input.isNullOrBlank()) {
            throw IllegalArgumentException(예외 메시지...)
        }
```

Reply:
Strings.isNullOrBlank() 라는 메서드도 있었군요!  
덕분에 코드가 훨씬 간결해졌네요😆